scripts/build/termux_step_setup_variables.sh

```diff
@@ -136,6 +125,9 @@ termux_step_setup_variables() {
 	TERMUX_PKG_BUILDDIR=$TERMUX_TOPDIR/$TERMUX_PKG_NAME/build
 	TERMUX_PKG_BUILD_DEPENDS=""
 	TERMUX_PKG_BUILD_IN_SRC=false
+	TERMUX_PKG_BUILD32=false # cross compilation (compilation of 32-bit packages for 64-bit devices)
+	TERMUX_PKG_ONLY_BUILD32=false # Specifies that the package is compiled only via cross compilation. Enabled automatically if cross compilation is enabled and the `TERMUX_PKG_EXCLUDED_ARCHES` variable contains `arm` and `i686` values.
```

Comment:
> Enabled automatically if cross compilation is enabled

Looks like it is only enabled if TERMUX_BUILD_ON_DEVICE = true (correct me if I am wrong), so this is not quite true

Reply:
No, the `TERMUX_PKG_ONLY_BUILD32` variable is enabled automatically if 32-bit compilation is enabled (i.e. `TERMUX_PKG_BUILD32=true`) and if the `ARM` and `i686` architectures are specified in the `TERMUX_PKG_EXCLUDED_ARCHES` variable. The `TERMUX_BUILD_ON_DEVICE` variable does not play any role in this.

PS: when checking, I noticed that the algorithm for automatically enabling `TERMUX_PKG_ONLY_BUILD32` is not implemented correctly; instead of checking `TERMUX_PKG_BUILD32`, the algorithm checks `TERMUX_BUILD_ON_DEVICE`. I'll fix it now.